packages/grid/stories/grid.stories.tsx

```diff
@@ -411,4 +423,47 @@ storiesOf('New Grid', module)
       height={ document.documentElement.clientHeight - 100 }
       onSortChanged={ (cols) => console.table(cols) }
     />
-  ));
+  ))
+  .add('Dinamic size + toggler should fit last column', () => {
```

Comment:
Not so critical but if you stand on input and press delete (change value from 3 to empty) you'll get the error.